src/styles.scss

```diff
@@ -104,7 +104,28 @@ h5 {
     display: inline !important;
 }
 
+mat-icon {
+    margin-bottom: 3px;
+    vertical-align: middle !important;
+}
+
+/* Generic custom styles */
+
+.contentButton {
+    margin: 0 5px !important;
+    border: 1px solid mat-color($rmm-default-primary) !important;
+    padding: 5px 5px 3px 5px !important;
+    line-height: 1.5em !important;
+    text-decoration: none !important;
+    white-space: nowrap;
+}
+
+.mainTextContent {
+    padding: 3em;
+}
 
+.contentButton mat-icon {
```

Comment:
What does an empty block like this do? Does it "reset" the things set in the regular `.contentButton`?